package.json

```
@@ -7,7 +7,9 @@
   "author": "Jani Eväkallio",
   "license": "MIT",
   "dependencies": {
-    "@jest/reporters": "^26.6.2",
     "chalk": "^4.1.0"
+  },
+  "peerDependencies": {
+    "@jest/reporters": "^25.0.0"
```

```suggestion
    "@jest/reporters": "^25.1.0 || ^26.0.0"
```

25.1.0 was first stable version of 25 (a version that was supposed to be a pre-release may or may not have been tagged as `latest`. `npm` is hard sometimes 👼).

Also, should still support v26